ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -45,12 +50,22 @@ public static void setup() throws ExtendedMotorFailureException {
         leftInput.send(leftMotors.addRamping(0.1f, FRC.constantPeriodic));
         rightInput.send(rightMotors.addRamping(0.1f, FRC.constantPeriodic));
 
+        FloatCell fastestDriveSpeed = new FloatCell();
+        ticksPerSecond.onChange().send(() -> {
+            if (Math.abs(feetPerSecond.get()) > fastestDriveSpeed.get()) {
+                fastestDriveSpeed.set(Math.abs(ticksPerSecond.get()));
+            }
+        });
```

Comment:
Better to use `ticksPerSecond.send((new_value) -> { ... new value ... });` than `ticksPerSecond.onChange().send(() -> { ... ticksPerSecond.get() ... });`